pkg/client/jiva/replica_client.go

```diff
@@ -113,3 +114,29 @@ func (c *ReplicaClient) MarkDiskAsRemoved(disk string) error {
 		Name: disk,
 	}, nil)
 }
+
+// GetStatus is the helper function for mayactl.It is used to get the response of
+// the replica created in json format and then the response is then decoded to
+// the desired structure.
+func (c *ReplicaClient) GetVolumeStats(address string, obj interface{}) (int, error) {
```

Comment:
Can the desired structure be a go template? Where the go template is a tabular format & this template is given to the stdoutput

Reply:
@AmitKumarDas no, we need to convert json into human readable format then it can be given to stdout. Any thoughts ?

Reply:
No @AmitKumarDas the main use of structures here to do some manipulation, and i don't know how to do those manipulations in templates.